Sources/StreamChat/WebSocketClient/Events/Event.swift

```diff
@@ -15,7 +15,7 @@ protocol EventWithPayload: Event {
 }
 
 /// A protocol for any `ChannelEvent` where it has a  `channel` payload.
-protocol ChannelSpecificEvent: EventWithPayload {
+protocol ChannelSpecificEvent: Event {
```

Comment:
The only place where it is used is `ChatChannelController` and it seems too much to have a separate type and make multiple events implement it for a single internal use-case. Would it be better to just list all the events that have `cid` instead, I don't know. 🤔